scripts/globals/items/federation_signet_staff.lua

```diff
@@ -4,9 +4,29 @@
 -- Effect: Signet
 -----------------------------------------
 require("scripts/globals/status");
+require("scripts/globals/conquest");
+require("scripts/globals/zone");
+require("scripts/globals/msg");
 -----------------------------------------
 
-function onItemCheck(target)
+function onItemCheck(target, caster)
+   
+    if (target:getNation() ~= NATION_WINDURST) then
+        return msgBasic.ITEM_CANNOT_USE_ON
+    end
+    
+    -- If target's current region is not a conquest region or not a nation city involved with conquest
+    if (target:getCurrentRegion() > REGION_JEUNO) then
```

Comment:
I am not sure, but retail might actually let you waste the charge and give yourself signet in an area it doesn't do anything with (nothings tops you from traipsing around ToAH with signet on, you just don't get CP/xtals/etc there)

Reply:
I was just going off wiki that says you can only use it in areas with conquest/regions.
http://ffxiclopedia.wikia.com/wiki/Kingdom_Signet_Staff

If we get retail proof that ffxiclopedia is wrong I'm down to remove that.

Reply:
"I got  source for it teo <link>" 

is sufficient

Reply:
What about <3 you teo whilst I show you some leg :p